mfr/extensions/utils.py

```diff
@@ -0,0 +1,11 @@
+from urllib.parse import parse_qs, urlencode
+
+from mfr.extensions import settings
+
+def development_export_url(urlparsed_url):
+        query_dict = parse_qs(urlparsed_url.query, keep_blank_values=True)
+        query_dict.pop('mode', None)
```

Comment:
It's been quite a while since I worked on this and don't recall off the top of my head. In brief testing I was not able to come up with a good answer other then that it isn't needed. I can remove the pop, but it's very possible I had a good reason that I no longer recall.

Reply:
This only affects local development. We can keep it there with the `TODO` note for you to recall the reason. :D